Alignment/TrackerAlignment/plugins/TrackerSystematicMisalignments.cc

```diff
@@ -210,9 +210,9 @@ void TrackerSystematicMisalignments::analyze(const edm::Event& event, const edm:
   if (!poolDbService.isAvailable())  // Die if not available
     throw cms::Exception("NotAvailable") << "PoolDBOutputService not available";
 
-  poolDbService->writeOne<Alignments>(&(*myAlignments), poolDbService->beginOfTime(), theAlignRecordName);
-  poolDbService->writeOne<AlignmentErrorsExtended>(
-      &(*myAlignmentErrorsExtended), poolDbService->beginOfTime(), theErrorRecordName);
+  poolDbService->writeOneIOV<Alignments>(myAlignments, poolDbService->beginOfTime(), theAlignRecordName);
```

Comment:
Isnt this missing a *?
```suggestion
  poolDbService->writeOneIOV<Alignments>(*myAlignments, poolDbService->beginOfTime(), theAlignRecordName);
```